src/homepageExperience/components/steps/arduino/InitializeClient.tsx

```diff
@@ -177,7 +177,12 @@ export const InitializeClient: FC<OwnProps> = ({
   return (
     <>
       <h1>Initialize Client</h1>
-      <p>Select or create a bucket to initialize the arduino client with.</p>
+      <h2 className="large-margins">Select or Create a bucket</h2>
```

Comment:
I think we Can we make the font weight match the `h1` tag? 

[Comparing these two examples, it’s immediately apparent why typographic hierarchy is so important.](https://www.toptal.com/designers/typography/typographic-hierarchy#:~:text=comparing%20these%20two%20examples%2C%20it%E2%80%99s%20immediately%20apparent%20why%20typographic%20hierarchy%20is%20so%20important)